<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1619</width>
    <height>916</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton:hover { color: red }
QCheckBox:hover:checked { color: red }</string>
  </property>
  <widget class="QComboBox" name="tl_bar_Box">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>30</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 7px;
background-color: white;
border: 2px solid #4CAF50;
font-size: 19px;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="map_Button">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 7px;
background-color: white;
border: 2px solid #4CAF50;
font-size: 19px;

</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QWebEngineView" name="webview" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>1521</width>
     <height>771</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>8</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QComboBox" name="province_Box">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>30</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 7px;
background-color: white;
border: 2px solid #4CAF50;
font-size: 19px;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="wordcloud_line_Box">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>30</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 7px;
background-color: white;
border: 2px solid #4CAF50;
font-size: 19px;</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>1521</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>PySide2.QtWebEngineWidgets</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
